src/components/Tasks.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import Checkbox from './Checkbox';
+import './Tasks.css';
+
+export default function Tasks({ task }) {
+
+  const dispatch = useDispatch();
+
+  const deleteTask = () => {
+
+    dispatch({type: 'REMOVE TASK', task: task.id});
```

Comment:
for the future projects:

It's good practice to move all Action Type/Name in a separate file:
export const REMOVE_TASK = 'REMOVE TASK';

https://redux.js.org/usage/reducing-boilerplate#actions

and then use as variable:
dispatch({type: REMOVE_TASK task: task.id});